### DISCUSSION TITLE
New feature

### DISCUSSION THREAD

**Discussion by hulchenko** [community/community#18385] [Other Feature Feedback, Questions, & Ideas] [answered]
Do you think that GitHub achievements is a good way to encourage its community to engage with more services?

**Comment by pRostok** [comment, marked as answer]
I have a generally positive view of this. I doubt the simpler-to-achieve badges will make the tougher-to-achieve ones seem less cool because you earn the difficult ones with more work. But there will be trial-and-error for sure.
